src/test/java/com/cloudbees/hudson/plugins/folder/FolderTest.java

```diff
@@ -137,22 +138,27 @@
         JenkinsRule.WebClient wc = r.createWebClient();
 
         // "foo" should copy "child"
-        copyFromGUI(f, wc, "foo", "xyz");
+        copyViaHttp(f, wc, "foo", "xyz");
```

Comment:
Code looks good, but was this copied by GUI to test the GUI itself or just as something necessary to test anything else?